model/encoder.py

```diff
@@ -22,7 +22,7 @@ def __init__(self, encoder_name, enc_normalizer,  is_change_sp_tokens=True):
         self.enc_normalizer = enc_normalizer
         self.is_change_sp_tokens = is_change_sp_tokens
         self.tokenizer = AutoTokenizer.from_pretrained(encoder_name)
-        
+        self.register_buffer("zero_emb", torch.zeros((self.encoder.config.hidden_size)))
```

Comment:
I'm not sure, but there might appear a broadcasting error, maybe registration should look like this
But if you executed the code, this correction is not relevant
```suggestion
        self.register_buffer("zero_emb", torch.zeros((1, self.encoder.config.hidden_size)))
```